Sources/SymbolKit/SymbolGraph/Module.swift

```diff
@@ -13,6 +13,14 @@ import Foundation
 extension SymbolGraph {
     /// A ``Module-swift.struct``  describes the module from which the symbols were extracted..
     public struct Module: Codable {
+        public enum CodingKeys: CodingKey {
+            case name
+            case bystanders
+            case platform
+            case version
+            case isVirtual
+        }
```

Comment:
Do we need this definition? Can we rely on the synthesised `CodingKeys` instead?

Reply:
I guess not.

Reply:
Fixed.